var/spack/repos/builtin/packages/py-adios/package.py

```diff
@@ -60,10 +58,12 @@ def setup_file(self):
             return 'setup.py'
 
     def build_clib(self, spec, prefix):
-        # calls: make [MPI=y] python
-        args = ''
+        rm = which('rm')
```

Comment:
Let's use `os.remove` instead to make this more portable

Reply:
Sure! (although ADIOS1 is Unix only :D )